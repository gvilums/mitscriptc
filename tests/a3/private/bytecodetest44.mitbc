function
{
	functions =
	[
		function
		{
			functions = [],
			constants = [],
			parameter_count = 1,
			local_vars = [],
			local_ref_vars = [],
			free_vars = [],
			names = [],
			instructions = 
			[
			]
		},
		function
		{
			functions = [],
			constants = [],
			parameter_count = 0,
			local_vars = [],
			local_ref_vars = [],
			free_vars = [],
			names = [],
			instructions = 
			[
			]
		},
		function
		{
			functions = [],
			constants = [],
			parameter_count = 1,
			local_vars = [],
			local_ref_vars = [],
			free_vars = [],
			names = [],
			instructions = 
			[
			]
		}
	],
	constants = [0, 100000, 1],
	parameter_count = 0,
	local_vars = [],
	local_ref_vars = [],
	free_vars = [],
	names = [aa, ba, ca, ab, bb, cb, ac, bc, cc, aaa, baa, caa, aba, bba, cba, aca, bca, cca, aab, bab, cab, abb, bbb, cbb, acb, bcb, ccb, ata, bta, cta, atb, btb, ctb, atc, btc, ctc, tcb, gcc, aasa, basa, casa, absa, bbsa, cbsa, acda, bsca, ccsa, aadb, badb, cadb, abdb, bbdb, cfdbb, acddb, bdcb, sdccb, i, print, input, intcast],
	instructions = 
	[
		load_func	0
		alloc_closure	0
		store_global	57
		load_func	1
		alloc_closure	0
		store_global	58
		load_func	2
		alloc_closure	0
		store_global	59
		load_const	0
		store_global	0
		load_const	0
		store_global	1
		load_const	0
		store_global	2
		load_const	0
		store_global	3
		load_const	0
		store_global	4
		load_const	0
		store_global	5
		load_const	0
		store_global	6
		load_const	0
		store_global	7
		load_const	0
		store_global	8
		load_const	0
		store_global	0
		load_const	0
		store_global	1
		load_const	0
		store_global	2
		load_const	0
		store_global	3
		load_const	0
		store_global	4
		load_const	0
		store_global	5
		load_const	0
		store_global	6
		load_const	0
		store_global	7
		load_const	0
		store_global	8
		load_const	0
		store_global	9
		load_const	0
		store_global	10
		load_const	0
		store_global	11
		load_const	0
		store_global	12
		load_const	0
		store_global	13
		load_const	0
		store_global	14
		load_const	0
		store_global	15
		load_const	0
		store_global	16
		load_const	0
		store_global	17
		load_const	0
		store_global	18
		load_const	0
		store_global	19
		load_const	0
		store_global	20
		load_const	0
		store_global	21
		load_const	0
		store_global	22
		load_const	0
		store_global	23
		load_const	0
		store_global	24
		load_const	0
		store_global	25
		load_const	0
		store_global	26
		load_const	0
		store_global	27
		load_const	0
		store_global	28
		load_const	0
		store_global	29
		load_const	0
		store_global	30
		load_const	0
		store_global	31
		load_const	0
		store_global	32
		load_const	0
		store_global	33
		load_const	0
		store_global	34
		load_const	0
		store_global	35
		load_const	0
		store_global	0
		load_const	0
		store_global	28
		load_const	0
		store_global	29
		load_const	0
		store_global	30
		load_const	0
		store_global	31
		load_const	0
		store_global	36
		load_const	0
		store_global	33
		load_const	0
		store_global	34
		load_const	0
		store_global	37
		load_const	0
		store_global	38
		load_const	0
		store_global	39
		load_const	0
		store_global	40
		load_const	0
		store_global	41
		load_const	0
		store_global	42
		load_const	0
		store_global	43
		load_const	0
		store_global	44
		load_const	0
		store_global	45
		load_const	0
		store_global	46
		load_const	0
		store_global	47
		load_const	0
		store_global	48
		load_const	0
		store_global	49
		load_const	0
		store_global	50
		load_const	0
		store_global	51
		load_const	0
		store_global	52
		load_const	0
		store_global	53
		load_const	0
		store_global	54
		load_const	0
		store_global	55
		load_const	0
		store_global	56
		load_global	56
		load_const	1
		swap
		gt
		not
		if	78
		load_global	0
		load_const	2
		add
		store_global	0
		load_global	48
		load_const	2
		add
		store_global	48
		load_global	55
		load_const	2
		add
		store_global	55
		load_global	47
		load_const	2
		add
		store_global	47
		load_global	37
		load_const	2
		add
		store_global	37
		load_global	34
		load_const	2
		add
		store_global	34
		load_global	4
		load_const	2
		add
		store_global	4
		load_global	8
		load_const	2
		add
		store_global	8
		load_global	31
		load_const	2
		add
		store_global	31
		load_global	33
		load_const	2
		add
		store_global	33
		load_global	14
		load_const	2
		add
		store_global	14
		load_global	15
		load_const	2
		add
		store_global	15
		load_global	16
		load_const	2
		add
		store_global	16
		load_global	17
		load_const	2
		add
		store_global	17
		load_global	18
		load_const	2
		add
		store_global	18
		load_global	19
		load_const	2
		add
		store_global	19
		load_global	15
		load_const	2
		add
		store_global	20
		load_global	48
		load_const	2
		add
		store_global	21
		load_global	56
		load_const	2
		add
		store_global	56
		goto	-82
		load_global	57
		load_global	0
		call	1
		pop
		load_global	57
		load_global	1
		call	1
		pop
		load_global	57
		load_global	2
		call	1
		pop
		load_global	57
		load_global	3
		call	1
		pop
		load_global	57
		load_global	4
		call	1
		pop
		load_global	57
		load_global	5
		call	1
		pop
		load_global	57
		load_global	6
		call	1
		pop
		load_global	57
		load_global	7
		call	1
		pop
		load_global	57
		load_global	8
		call	1
		pop
		load_global	57
		load_global	0
		call	1
		pop
		load_global	57
		load_global	1
		call	1
		pop
		load_global	57
		load_global	2
		call	1
		pop
		load_global	57
		load_global	3
		call	1
		pop
		load_global	57
		load_global	4
		call	1
		pop
		load_global	57
		load_global	5
		call	1
		pop
		load_global	57
		load_global	6
		call	1
		pop
		load_global	57
		load_global	7
		call	1
		pop
		load_global	57
		load_global	8
		call	1
		pop
		load_global	57
		load_global	9
		call	1
		pop
		load_global	57
		load_global	10
		call	1
		pop
		load_global	57
		load_global	11
		call	1
		pop
		load_global	57
		load_global	12
		call	1
		pop
		load_global	57
		load_global	13
		call	1
		pop
		load_global	57
		load_global	14
		call	1
		pop
		load_global	57
		load_global	15
		call	1
		pop
		load_global	57
		load_global	16
		call	1
		pop
		load_global	57
		load_global	17
		call	1
		pop
		load_global	57
		load_global	18
		call	1
		pop
		load_global	57
		load_global	19
		call	1
		pop
		load_global	57
		load_global	20
		call	1
		pop
		load_global	57
		load_global	21
		call	1
		pop
		load_global	57
		load_global	22
		call	1
		pop
		load_global	57
		load_global	23
		call	1
		pop
		load_global	57
		load_global	24
		call	1
		pop
		load_global	57
		load_global	25
		call	1
		pop
		load_global	57
		load_global	26
		call	1
		pop
		load_global	57
		load_global	27
		call	1
		pop
		load_global	57
		load_global	28
		call	1
		pop
		load_global	57
		load_global	29
		call	1
		pop
		load_global	57
		load_global	30
		call	1
		pop
		load_global	57
		load_global	31
		call	1
		pop
		load_global	57
		load_global	32
		call	1
		pop
		load_global	57
		load_global	33
		call	1
		pop
		load_global	57
		load_global	34
		call	1
		pop
		load_global	57
		load_global	35
		call	1
		pop
		load_global	57
		load_global	0
		call	1
		pop
		load_global	57
		load_global	28
		call	1
		pop
		load_global	57
		load_global	29
		call	1
		pop
		load_global	57
		load_global	30
		call	1
		pop
		load_global	57
		load_global	31
		call	1
		pop
		load_global	57
		load_global	36
		call	1
		pop
		load_global	57
		load_global	33
		call	1
		pop
		load_global	57
		load_global	34
		call	1
		pop
		load_global	57
		load_global	37
		call	1
		pop
		load_global	57
		load_global	38
		call	1
		pop
		load_global	57
		load_global	39
		call	1
		pop
		load_global	57
		load_global	40
		call	1
		pop
		load_global	57
		load_global	41
		call	1
		pop
		load_global	57
		load_global	42
		call	1
		pop
		load_global	57
		load_global	43
		call	1
		pop
		load_global	57
		load_global	44
		call	1
		pop
		load_global	57
		load_global	45
		call	1
		pop
		load_global	57
		load_global	46
		call	1
		pop
		load_global	57
		load_global	47
		call	1
		pop
		load_global	57
		load_global	48
		call	1
		pop
		load_global	57
		load_global	49
		call	1
		pop
		load_global	57
		load_global	50
		call	1
		pop
		load_global	57
		load_global	51
		call	1
		pop
		load_global	57
		load_global	52
		call	1
		pop
		load_global	57
		load_global	53
		call	1
		pop
		load_global	57
		load_global	54
		call	1
		pop
		load_global	57
		load_global	55
		call	1
		pop
	]
}