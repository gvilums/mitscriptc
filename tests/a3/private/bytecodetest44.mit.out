100000
0
0
0
100000
0
0
0
100000
100000
0
0
0
100000
0
0
0
100000
0
0
0
0
0
100000
100000
100000
100000
100000
100000
100001
100001
0
0
0
0
0
0
0
0
0
100000
0
100000
100000
0
100000
0
0
0
100000
0
100000
100000
100000
0
0
0
0
0
0
0
0
0
100000
100000
0
0
0
0
0
0
100000
